projects/v2/src/app/json-typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.json' {
```

Comment:
No. The JSON data should be loaded dynamically at runtime via fetch/httpclient from assets.

Reply:
ok got it